python/HeaderService/hutils.py

```diff
@@ -192,13 +194,11 @@ def get_date_utc(timeStamp=None, format='isot'):
 
     """
 
-    from astropy.time import Time
-    from datetime import datetime
     if timeStamp is None:
-        utc_time = (datetime.utcnow()).isoformat()
+        utc_time = (datetime.datetime.utcnow()).isoformat()
     else:
-        utc_time = datetime.utcfromtimestamp(timeStamp).isoformat()
-    t = Time(utc_time, format=format, scale='utc')
+        utc_time = datetime.datetime.utcfromtimestamp(timeStamp).isoformat()
```

Comment:
You can avoid the unnecessary stringification here by going directly to Astropy Time:
```
t = astropy.time.Time(timeStamp, format="unix")
```

Reply:
Thanks for the suggestion, it's been implemented.